### ISSUE TITLE
Registration success page

### ISSUE THREAD

**Issue by ksmacleod99** [OperationCode/front-end#771]
# Feature request
In order to improve the on-boarding/registration workflow, could we create a 'Registration Success' page before the user is redirected to their profile?

## Proposed solution
A page that has next steps such as:
"Check your email for an invitation to our Slack Community (or click here)"
"What is Slack?" 
"Go to your profile page"
"Check out some resources" 

## Potential alternative solutions
A temporary banner (or an escapable banner) on their profile.
A timed redirect message component, i.e. "Congrats! You are now being redirected..." or something to that effect. 

## Additional context
Currently, the registration process ends with the user finishing the workflow and being redirected to their profile page. This might need some designing, but I don't imagine it has to be too complicated.

**Comment by calincionca35** [comment]
Wording suggestion for the displayed message:
"You were successful in creating your account! Please check your email for your invitation to our Slack channel!"

**Comment by DEBSUBHRO** [comment]
Can I propose a mockup?

**Comment by C-Lion** [comment]
What format is preferred for the mockup? PDF? XML?

**Comment by kylemh** [comment]
Just a screenshot here is fine. The PR itself is posted for work to be complete regarding development.

**Comment by kylemh** [comment]
From @C-Lion:

https://user-images.githubusercontent.com/14170817/67628481-de3ad780-f86e-11e9-876f-cbf06beef533.png

**Comment by C-Lion** [comment]
The issue labels are for a design/mock only. If you want the page built, it should indicate that in the issue description & indicate where it should be placed and probably use the correct labels so people know what you really want done.

**Comment by kylemh** [comment]
The labels just indicate work that's required before the ticket is resolvable. If you hover over the label it will say: "The feature requires design input and/or mocks."

You can only resolve issues in this (and most repositories) with code, unless otherwise specified.

**Comment by cbituin** [comment]
Has there been any progress or any other mocks submitted on this?

**Comment by kylemh** [comment]
Nope. You’re welcome to snag!

**Comment by joshuadempsey** [comment]
Does this issue still need worked on? I'd be happy to try and help. New to GH but I'm a competent(ish) web-dev.

**Comment by kylemh** [comment]
Shoot for the stars, Joshua! You can DM me on our Slack team if you need anything.

**Comment by LindaCBeall** [comment]
@datphotogurl any comments on this?
https://xd.adobe.com/view/2d58a230-a8e8-4dc0-ad61-2cab6c69b002-780b/
<img width="1281" alt="Screen Shot 2020-10-07 at 2 20 48 PM" src="https://user-images.githubusercontent.com/37910748/95389236-63fd3a80-08a8-11eb-9442-fa1d2ea6580d.png">

**Comment by calincionca35** [comment]
For Step 3, I would like to suggest, suggesting that, if applicable, the new member consider adding links to GitHub and LinkedIn accounts.

**Comment by datphotogurl** [comment]
@LindaCBeall I like the concept and all the components advising the new member of where to go next. We just need to polish up, simplify the interface for ease of legibility and ensure the mock-ups include a desktop view and mobile view. We also need a one-liner to let people know why our community connects on Slack. Lemme post that on OC-Tech for feedback and approval, then I will add here after approval.  I think that artboard for iPad Pro on the right isn't part of the design concept right?

**Comment by LindaCBeall** [comment]
Ok, sounds good. No, the stuff to the right is just notes and an alternative to the "Other Resources to Check Out" accordion component if we want something more visual.

**Comment by recondesigns** [comment]
I am curious to see what I can do here, so I got a few questions and discussion.

> In order to improve the on-boarding/registration workflow, could we create a 'Registration Success' page before the user is redirected to their profile?

1. The ticket is requesting we _create_ a Registration Success page. One currently exists and pretty much includes what is in under "Proposed Solution" (screenshot). I am curious if the ticket still needs to be open?
2. I just went through the registration process, and I was redirected to the Registration Success page and no "profile". Are we not using a profile to redirect folks to?
